test/core/libp2p/multi/multihash_test.cpp

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gtest/gtest.h>
+#include "libp2p/multi/multihash.hpp"
+
+using kagome::expected::Error;
+using kagome::expected::Result;
+using kagome::expected::Value;
+using libp2p::multi::Multihash;
+
+using kagome::common::Buffer;
+
+/**
+ *   @given a buffer with a hash
+ *   @when creating a multihash using the buffer
+ *   @then a correct multihash object is created if the hash size is not greater
+ *than maximum length
+ **/
+TEST(Multihash, Multihash_Create) {
```

Comment:
```suggestion
TEST(Multihash, Create) {
```
It's already pretty understandable, that test is about Multihash